DOCS.md

```diff
@@ -171,12 +171,13 @@ The list of all understood properties is given below.
 | gridColor                  | string or int           | `0x787878`   | can be HEX, RGB, or RGBA string or hexadecimal value                 | `false`    |
 | gridOpacity                | float                   | `1.0`        | must be in [`0`,`1`]                                                 | `false`    |
 | items                      | array                   | `[]`         | see [`data`](#data)                                                  | `false`    |
-| itemSize                   | int                     |              | number of pixels                                                     | `true`     |
+| itemSize                   | int                     |              | ignored when `cellSize` is defined                                                | `true`     |
```

Comment:
`itemSize` should not be ignored when `cellSize` is defined. What would we need it for then?